onnxruntime/python/tools/featurizer_ops/data_frame_tool.py

```diff
@@ -5,7 +5,9 @@
 # This script is an experimental feature. Use at your own risk.
 # Contributions are welcome.
 
+from datetime import datetime
 import numpy as np
+import pandas as pd
```

Comment:
> pandas as pd [](start = 6, length = 13)

Initially, not importing pandas was intentional to shift the dependency management to the client. But as it is, it is a client script and actually does not belong to ORT.